runner/run_bench.py

```diff
@@ -613,8 +657,8 @@ def send_to_slack_txt(txt):
 
 def send_to_slack_attachment(title, fields, text="", success=True):
     fields['Host'] = HOSTNAME
-    fields['Commit'] = RUN_DATA.current_commit[:6]
-    fields['Branch'] = REPO_BRANCH
+    fields['Commit'] = (RUN_DATA.current_commit or '')[:6]
```

Comment:
Under what circumstance would this not be initialized?

Reply:
I found that in certain cases an error can occur before initialization, in which case we try to send the stack trace to slack and then hit an exception here.